driver.go

```diff
@@ -404,9 +561,52 @@ func makeLabels(vars map[string]interface{}, types []string) string {
 	return out.String()
 }
 
-func makeProperties(vars map[string]interface{}, properties map[string]*ls.PropertyValue, idAndValue map[string]*ls.PropertyValue) string {
+// getNeo4jType will get native neo4j type based on given value to be represented in the database
+func getNeo4jType(node graph.Node, val interface{}, c Config) interface{} {
```

Comment:
You should not need this. What you need is: nativeValuetoNeo4jValue(interface{}) interface{}
It should convert types.Date/Time to neo4j.Date/Time